crates/frontend/src/components/input_components.rs

```diff
@@ -0,0 +1,70 @@
+use leptos::*;
+use serde_json::{Map, Value};
+
+use crate::components::dropdown::{Dropdown, DropdownBtnType, DropdownDirection};
+
+#[component]
+pub fn enum_dropdown(
+    schema: Map<String, Value>,
+    config_value: String,
+    handle_change: Callback<String, ()>,
+    #[prop(default = String::new())] class: String,
+    #[prop(default = false)] disabled: bool,
+) -> impl IntoView {
+    let mut enum_array: Vec<String> = vec![];
```

Comment:
@mahatoankitkumar lets not assume enum array , as array of string only , it can be integer or decimal also
can we have that support , because if I am declaring enum of number in UI form dropdown shows nothing